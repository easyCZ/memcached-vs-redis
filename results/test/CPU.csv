timestamp, CPU free
25, 99.94
26, 100.00
27, 100.00
28, 100.00
29, 100.00
30, 97.33
31, 90.65
32, 100.00
33, 100.00
34, 38.04
35, 30.80
36, 30.97
37, 30.45
38, 31.02
39, 30.80
40, 30.97
41, 30.90
42, 30.56
43, 30.90
44, 30.97
45, 30.85
46, 30.90
47, 30.45
48, 30.90
49, 30.97
50, 30.96
51, 30.74
52, 30.50
53, 19.35
54, 18.95
55, 19.03
56, 19.11
57, 18.83
58, 18.95
59, 18.99
60, 19.03
61, 18.99
62, 18.99
63, 18.91
64, 18.59
65, 18.95
66, 18.88
67, 19.07
68, 18.83
69, 19.16
70, 19.15
71, 18.91
72, 19.07
73, 18.95
74, 19.03
75, 19.08
76, 34.05
77, 58.29
78, 62.80
79, 62.57
80, 62.04
81, 62.80
82, 62.41
83, 62.69
84, 62.69
85, 62.20
86, 62.57
87, 62.64
88, 63.24
89, 79.65
90, 81.59
91, 82.27
92, 82.09
93, 82.12
94, 82.24
95, 82.09
96, 82.09
97, 82.09
98, 82.42
99, 82.24
100, 77.74
101, 87.06
102, 93.43
103, 100.00
104, 100.00
105, 99.50
106, 99.83
107, 100.00
108, 100.00
109, 100.00
110, 100.00
111, 99.67
112, 100.00
113, 100.00
114, 100.00
115, 100.00
116, 100.00
117, 95.38
118, 94.64
119, 100.00
120, 100.00
121, 100.00
122, 100.00
123, 99.50
124, 100.00
125, 100.00
126, 100.00
127, 100.00
128, 100.00
129, 99.67
130, 100.00
131, 100.00
132, 100.00
133, 100.00
134, 100.00
135, 99.67
136, 100.00
137, 100.00
138, 100.00
139, 100.00
140, 100.00
141, 99.67
142, 100.00
143, 100.00
144, 100.00
145, 100.00
146, 100.00
147, 99.67
148, 100.00
149, 100.00
150, 100.00
151, 100.00
152, 100.00
153, 99.50
154, 100.00
155, 100.00
156, 100.00
157, 100.00
158, 100.00
159, 99.50
160, 100.00
161, 99.83
162, 99.67
163, 100.00
164, 99.67
