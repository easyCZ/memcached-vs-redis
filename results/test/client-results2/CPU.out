Linux 3.13.0-66-generic (nsl200) 	09/11/15 	_x86_64_	(6 CPU)

09/11/15 09:31:25
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.01    0.01    0.04    0.00    0.00   99.94

09/11/15 09:31:26
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:27
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:28
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:29
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:30
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           2.17    0.00    0.50    0.00    0.00   97.33

09/11/15 09:31:31
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           5.18    0.00    3.84    0.33    0.00   90.65

09/11/15 09:31:32
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:33
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:31:34
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           9.47    0.00   52.50    0.00    0.00   38.04

09/11/15 09:31:35
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          12.11    0.00   57.09    0.00    0.00   30.80

09/11/15 09:31:36
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.07    0.00   57.96    0.00    0.00   30.97

09/11/15 09:31:37
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.59    0.00   57.61    0.35    0.00   30.45

09/11/15 09:31:38
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          10.92    0.00   58.06    0.00    0.00   31.02

09/11/15 09:31:39
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.76    0.00   57.27    0.17    0.00   30.80

09/11/15 09:31:40
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.42    0.00   57.61    0.00    0.00   30.97

09/11/15 09:31:41
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          10.94    0.00   58.16    0.00    0.00   30.90

09/11/15 09:31:42
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.98    0.00   57.12    0.35    0.00   30.56

09/11/15 09:31:43
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          12.85    0.00   56.25    0.00    0.00   30.90

09/11/15 09:31:44
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.94    0.00   57.09    0.00    0.00   30.97

09/11/15 09:31:45
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          10.92    0.00   58.23    0.00    0.00   30.85

09/11/15 09:31:46
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          12.85    0.00   56.25    0.00    0.00   30.90

09/11/15 09:31:47
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.94    0.00   57.27    0.35    0.00   30.45

09/11/15 09:31:48
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.63    0.00   57.47    0.00    0.00   30.90

09/11/15 09:31:49
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.07    0.00   57.96    0.00    0.00   30.97

09/11/15 09:31:50
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          12.70    0.00   56.35    0.00    0.00   30.96

09/11/15 09:31:51
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          12.61    0.00   56.65    0.00    0.00   30.74

09/11/15 09:31:52
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          10.92    0.00   58.23    0.35    0.00   30.50

09/11/15 09:31:53
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.32    0.00   65.32    0.00    0.00   19.35

09/11/15 09:31:54
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.52    0.00   65.52    0.00    0.00   18.95

09/11/15 09:31:55
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.38    0.00   65.59    0.00    0.00   19.03

09/11/15 09:31:56
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          17.51    0.00   63.38    0.00    0.00   19.11

09/11/15 09:31:57
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          17.00    0.00   63.97    0.20    0.00   18.83

09/11/15 09:31:58
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.13    0.00   64.92    0.00    0.00   18.95

09/11/15 09:31:59
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.35    0.00   65.66    0.00    0.00   18.99

09/11/15 09:32:00
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          17.21    0.00   63.77    0.00    0.00   19.03

09/11/15 09:32:01
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          14.55    0.00   66.46    0.00    0.00   18.99

09/11/15 09:32:02
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.16    0.00   64.85    0.00    0.00   18.99

09/11/15 09:32:03
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          14.49    0.00   66.60    0.00    0.00   18.91

09/11/15 09:32:04
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          14.55    0.00   66.46    0.40    0.00   18.59

09/11/15 09:32:05
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          14.11    0.00   66.94    0.00    0.00   18.95

09/11/15 09:32:06
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.26    0.00   65.86    0.00    0.00   18.88

09/11/15 09:32:07
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.63    0.00   64.30    0.00    0.00   19.07

09/11/15 09:32:08
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          13.97    0.00   67.21    0.00    0.00   18.83

09/11/15 09:32:09
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.57    0.00   64.27    0.00    0.00   19.16

09/11/15 09:32:10
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.52    0.00   65.32    0.00    0.00   19.15

09/11/15 09:32:11
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.90    0.00   64.99    0.20    0.00   18.91

09/11/15 09:32:12
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.23    0.00   64.71    0.00    0.00   19.07

09/11/15 09:32:13
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.73    0.00   64.31    0.00    0.00   18.95

09/11/15 09:32:14
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          16.80    0.00   64.17    0.00    0.00   19.03

09/11/15 09:32:15
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          15.66    0.00   65.26    0.00    0.00   19.08

09/11/15 09:32:16
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
          11.35    0.00   54.59    0.00    0.00   34.05

09/11/15 09:32:17
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           8.01    0.00   33.70    0.00    0.00   58.29

09/11/15 09:32:18
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.48    0.00   29.72    0.00    0.00   62.80

09/11/15 09:32:19
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           8.01    0.00   29.42    0.00    0.00   62.57

09/11/15 09:32:20
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.96    0.00   29.63    0.37    0.00   62.04

09/11/15 09:32:21
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.85    0.00   29.35    0.00    0.00   62.80

09/11/15 09:32:22
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.59    0.00   30.00    0.00    0.00   62.41

09/11/15 09:32:23
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           8.58    0.00   28.73    0.00    0.00   62.69

09/11/15 09:32:24
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.65    0.00   29.66    0.00    0.00   62.69

09/11/15 09:32:25
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           6.70    0.00   30.73    0.37    0.00   62.20

09/11/15 09:32:26
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           8.01    0.00   29.42    0.00    0.00   62.57

09/11/15 09:32:27
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           7.81    0.00   29.55    0.00    0.00   62.64

09/11/15 09:32:28
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           8.09    0.00   28.68    0.00    0.00   63.24

09/11/15 09:32:29
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           4.25    0.00   16.11    0.00    0.00   79.65

09/11/15 09:32:30
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           2.48    0.00   15.58    0.35    0.00   81.59

09/11/15 09:32:31
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.55    0.00   14.18    0.00    0.00   82.27

09/11/15 09:32:32
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.01    0.00   14.89    0.00    0.00   82.09

09/11/15 09:32:33
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.54    0.00   14.34    0.00    0.00   82.12

09/11/15 09:32:34
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           4.09    0.00   13.68    0.00    0.00   82.24

09/11/15 09:32:35
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.55    0.00   14.18    0.18    0.00   82.09

09/11/15 09:32:36
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.90    0.00   14.01    0.00    0.00   82.09

09/11/15 09:32:37
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.55    0.00   14.36    0.00    0.00   82.09

09/11/15 09:32:38
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           2.84    0.00   14.74    0.00    0.00   82.42

09/11/15 09:32:39
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           3.20    0.00   14.56    0.00    0.00   82.24

09/11/15 09:32:40
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           4.85    0.00   17.06    0.36    0.00   77.74

09/11/15 09:32:41
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           2.66    0.00   10.28    0.00    0.00   87.06

09/11/15 09:32:42
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           1.90    0.00    4.67    0.00    0.00   93.43

09/11/15 09:32:43
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:44
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:45
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.50    0.00   99.50

09/11/15 09:32:46
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.17    0.00    0.00   99.83

09/11/15 09:32:47
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:48
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:49
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:50
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:51
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

09/11/15 09:32:52
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:53
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:54
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:55
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:56
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:32:57
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           1.54    0.00    2.74    0.34    0.00   95.38

09/11/15 09:32:58
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           1.73    0.00    3.63    0.00    0.00   94.64

09/11/15 09:32:59
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:00
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:01
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:02
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:03
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.50    0.00   99.50

09/11/15 09:33:04
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:05
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:06
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:07
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:08
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:09
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

09/11/15 09:33:10
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:11
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:12
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:13
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:14
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:15
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

09/11/15 09:33:16
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:17
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:18
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:19
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:20
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:21
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

09/11/15 09:33:22
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:23
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:24
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:25
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:26
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:27
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

09/11/15 09:33:28
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:29
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:30
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:31
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:32
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:33
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.50    0.00   99.50

09/11/15 09:33:34
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:35
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:36
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:37
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:38
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:39
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.17    0.00    0.00    0.33    0.00   99.50

09/11/15 09:33:40
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:41
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.17    0.00    0.00   99.83

09/11/15 09:33:42
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.17    0.00    0.17    0.00    0.00   99.67

09/11/15 09:33:43
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.00    0.00  100.00

09/11/15 09:33:44
avg-cpu:  %user   %nice %system %iowait  %steal   %idle
           0.00    0.00    0.00    0.33    0.00   99.67

